src/draive/helpers/env.py

```diff
@@ -89,3 +90,35 @@ def getenv_str(
         return value
     else:
         return default
+
+
+def load_env(
+    path: str | None = None,
+    override: bool = True,
+) -> None:
+    # minimal implementation
+    # allowing only subset of formatting:
+    # lines starting with '#' are ignored
+    # other comments are not allowed
+    # each element must be a `key=value` pair
+    # without whitespaces or additional characters
+    # keys without values are ignored
+    with open(file=path or ".env") as file:
+        for line in file.readlines():
+            if line.startswith("#"):
+                continue  # ignore commented
+
+            idx: int  # find where key ends
+            for element in enumerate(line):
+                if element[1] == "=":
```

Comment:
Why not using string `split("=")` metod?

Reply:
there might be multiple occurences of "=" - using split is risky as only the first occurrence splits value from key